src/tasks/sync.js

```diff
@@ -149,7 +149,10 @@ const SyncSpaces = {
     for (const folder of sourceFolders) {
       try {
         const folderResult = await this.client.get(`spaces/${this.sourceSpaceId}/stories/${folder.id}`)
-        const { data } = await this.client.get(`spaces/${this.targetSpaceId}/stories`, { with_slug: folder.full_slug })
+        const { data } = await this.client.get(`spaces/${this.targetSpaceId}/stories`, {
+            with_slug: folder.full_slug,
```

Comment:
Hi, @geometricpanda can we substitute the ternary with a simple `if` for readability? Something like this would work

```js
const params = { with_slug: folder.full_slug };
if (this.startsWith) {
  params.starts_with = this.startsWith;
}

const { data } = await this.client.get(
  `spaces/${this.targetSpaceId}/stories`,
  params
);
```

Reply:
Sure thing. Was simply following the example further up. Always love moving towards better readability :-)

Reply:
hey @alvarosabu, i've made those changes - also added a bit extra to scope how much it was pulling from the source space